### ISSUE TITLE
explicitly name the metadata files that assemblyLine uses

### ISSUE THREAD

**Issue by mobb** [EDIorg/EMLassemblyline#37]
Today, the assemblyLine code looks for a set of metadata tables with exact names. 
It would be more transparent to name those metadata files somewhere. in the assemble EML script it might look like the below. It might be better in a config script that was sourced by the assemble_EML script.

```R 
...
file_path <- "."

user_id <- "mobrien"
user_domain <- 'EDI'
package_id <- "edi.437.1"

dataset_title <- 'TO DO: Dataset title'

# assemble resource-level metadata objects
abstract_metadata     <- 'abstract.docx'
methods_metadata      <- 'methods.docx'
geocov_metadata       <- 'geographic_coverage.txt'
intellrights_metadata <- 'intellectual_rights.txt'
keywords_metadata     <- 'keywords.txt'
people_metadata       <- 'personnel.txt'
# taxon_metadata        <- 'taxonomy.txt'

# other high level metadata not available in predefined objects
temp_cov <- c("2000-01-01", "2000-01-01")
maint_desc <- "completed"

# assemble data entities
# Entity file names.  if more than one, use a quoted, comma-separated list
file_names <- c('Synthesis_Master_2019_Marine.csv')

# if more than one, use a quoted, comma-sep list, same order as file_names
entity_descriptions <- c('Synthesis samples from 5 field locations for Synchrony Working Group (LTER)')

# attribute lists and defintions of categorical variables (metadata objects)
entity_attributes <- c('attributes_Synthesis_Master_2019_Marine.txt')
entity_catvars    <- c('catvars_Synthesis_Master_2019_Marine.txt')

make_eml( ...

```

**Comment by mobb** [comment]
Advantages:
1. assemblyLine knows exactly which metadata files to use. less chance of filename collisions.
1. a script which created the group of metadata templates could also create this chunk of code for the assemble_EML template.
1. if you wanted to add support for reading metadata in CSVs instead of tab-sep files, it would be obvious which one to use
1. later (e.g., when we want to add functionality to read metadata from other sources, it could replace this chunk.

**Comment by clnsmth** [comment]
This has been fixed in f1c0f74afa7945f8c21fea7406a1d35a12dee345.
